apps/worker/src/services/stats-calculator.ts

```
@@ -598,31 +597,44 @@ export async function calculateCurrentMinuteHistory() {
+}
+
 export async function calculateAggregatedStatistics() {
-	logger.debug("Starting 5-minute aggregated statistics calculation...");
+	logger.debug("Starting aggregated statistics calculation...");
 
 	try {
 		const database = db;
-		// eslint-disable-next-line no-mixed-operators
-		const fiveMinutesAgo = new Date(Date.now() - 5 * 60 * 1000);
+		const oneHourAgo = new Date(
+			Date.now() - ROUTING_WINDOW_MINUTES * 60 * 1000,
```

ESLint

[error] Unexpected mix of '-' and '*'. Use parentheses to clarify the intended order of operations.

(no-mixed-operators)